labs/lab4.sh

```diff
@@ -50,6 +50,13 @@ DOCKER_IMAGE="ghcr.io/infraguys/debian_lab"
 
 
 create() {
+    if [ -f /tmp/lab* ]; then
+        LABS=$(ls /tmp/lab*)
+        echo $(basename $LABS) should be deleted before start lab
```

Comment:
Для прозрачности давай уточним фразу

```suggestion
        echo "You didn't finish with previous lab!"
        echo Please run `$(basename $LABS) delete` to cleanup previous artifacts.
```

Вот тут как раз поможет если будем имя скрипта юзать вместо LAB_NAME